package/MDAnalysis/coordinates/base.py

```diff
@@ -101,17 +101,18 @@
    :members:
    :inherited-members:
 
-
 Converters
 ----------
+Converters output information to other libraries. 
 
-Converters output information to other libraries.
+.. deprecated:: 2.7.0
+    All converter code has been moved to :mod:`MDAnalysis.converters` and will
+    be removed in 3.0.0.
```

Comment:
Make clearer that converters still exist but will be removed from `coordinates`.
```suggestion
    be removed from the :mod:`MDAnalysis.coordinates.base` module in 3.0.0.
```
If we actually manage to move converters to kits for 3.0.0 then we can still rewrite this message.

Reply:
I think there's maybe a misconception here - we aren't moving the ConverterBase machinery out of core.

We still want to provide the necessary API for folks to create converters, it's just that new converter implementations (and maybe some old ones) will have to move out of the library to avoid library bloat.

Reply:
Thanks for the clarification.